.github/workflows/build-pr.yml

```
@@ -93,4 +93,36 @@ jobs:
     needs: [e2e-build-ios]
-    secrets: inherit
\ No newline at end of file
+    secrets: inherit
+
+  generate-version-code:
+    name: Generate Version Code
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)